hub/core/storage/tests/test_storage_provider.py

```diff
@@ -133,3 +134,4 @@ def test_pickling(storage):
     pickle_file = open("storage_pickle", "rb")
     unpickled_storage = pickle.load(pickle_file)
     assert unpickled_storage[FILE_1] == b"hello world"
+    os.remove("storage_pickle")
```

Comment:
```python
import click


#.....


@parametrize_all_storages
def test_pickling(storage):
    with click.CliRunner().isolated_filesystem():
        FILE_1 = f"{KEY}_1"
        storage[FILE_1] = b"hello world"
        assert storage[FILE_1] == b"hello world"
        pickle_file = open("storage_pickle", "wb")
        pickle.dump(storage, pickle_file)
        pickle_file = open("storage_pickle", "rb")
        unpickled_storage = pickle.load(pickle_file)
        assert unpickled_storage[FILE_1] == b"hello world"
```

This will make sure the temp files are deleted even if the test fails.

Reply:
Understood. Thank you for your input! I'll make these changes and push.